src/components/availability-label/usePhysicalAvailabilityData.ts

```diff
@@ -0,0 +1,74 @@
+import { useConfig } from "../../core/utils/config";
+import { FaustId } from "../../core/utils/types/ids";
+import useGetAvailability from "../../core/utils/useGetAvailability";
+import { ManifestationMaterialType } from "../../core/utils/types/material-type";
+
+const usePhysicalAvailabilityData = ({
+  enabled,
+  faustIds,
+  manifestText
+}: {
+  enabled: boolean;
+  faustIds: FaustId[] | null;
+  manifestText: string;
+}) => {
+  const config = useConfig();
+
+  const response = useGetAvailability({
+    faustIds: faustIds ?? [],
+    config,
+    options: {
+      query: {
+        // FBS / useGetAvailabilityV3 is responsible for handling availability
+        // for physical items. This will be the majority of all materials so we
+        // use this for everything except materials that are explicitly online.
+        enabled:
+          enabled &&
+          faustIds !== null &&
+          manifestText !== ManifestationMaterialType.article
+      }
+    }
+  });
+
+  const { isLoading, data } = response;
+
+  // If hook is not enabled make it clear that the loading and availability status is unknown.
+  if (!enabled) {
+    return {
+      isLoading: null,
+      isAvailable: null
+    };
+  }
+
+  // Articles are always available.
+  if (manifestText === ManifestationMaterialType.article) {
```

Comment:
Okay, so there is something that I think should probably be updated here (and also wherever we check for whether it is an article). 
Since this was made, we added new manifestation types that are considered an "article" - under the `ManifestationMaterialType`. There are these different types of articles - article, paperArticle, onlineArticle, and earticle. See picture below.
![image](https://github.com/user-attachments/assets/b09f407b-cbe6-4172-a509-4a0ca62c1954)

I believe all of these are always supposed to be available, so maybe we should introduce a helper function that checks whether the `manifestText` is any of these, instead of just the `ManifestationMaterialType.article` .

Reply:
It current code onlyManifestationMaterialType.article is being considered:
https://github.com/danskernesdigitalebibliotek/dpl-react/blob/develop/src/components/availability-label/helper.ts
Let's get this through and create a new task about this problem.

Reply:
We decided to fix it in current PR anyways...